docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+:_content-type: REFERENCE
+[id="mta-web-assessment-module_{context}"]
+= Assessment Module
+
+The assessment process for applications that are to be migrated in Red Hat Migration Toolkit for Applications (MTA) has been separated from other tool features and enhanced in version 7.0.0.
+
+* A separate new Assessment hub includes the Assessment module, replacing the older _Pathfinder_ feature.
+* Assessment supports multiple questionnaires, relevant to each assessed application.
+* Default questionnaires can be customized and saved.
+* User can design and version custom questionnaires and questions using YAML syntax.
+* Assessed applications are grouped into *archetypes* with similar application characteristics.
+* Assessment module integrates tightly with the application inventory.
+* Assessment questionnaires can be imported and exported.
+* Enhanced questionnaire capabilities:
+** Conditional questions
+** Application auto-tagging according to answers
+** Automated answers from tags in application archetypes
+
+In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved default and custom questionnaires.
+
+:_content-type: PROCEDURE
+[id="mta-assessment-start-questionnaire_{context}"]
+== Opening an assessment questionnaire
+
+The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions relevant to the application, such as dependencies, *[more question types]*. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.
+
+.To import an application
+
+// Get updated image for MTA
+image::mta-assessment-apps-01.png[MTA application inventory]
+
+. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires. 
+. Click *Create new*. A *New application* dialog appears.
+
+// Get updated image for MTA
+image::mta-assessment-new-app-01.png[MTA application inventory]
+
+[start=3]
+. In the form, enter the following information:
```

Comment:
"supply" or something similar is probably a better verb that "enter" since there are fields that a user completes by selecting one or more items. "enter" means "type."

Reply:
Accepted